config/core.entity_view_display.media.image.easy_reponsive_images.yml

```diff
@@ -0,0 +1,32 @@
+uuid: 4c870480-5d94-43ee-b738-55904b1c108d
```

Comment:
Do we need this display?  Since we have all the other entity view displays based on the aspect ratios, I am not sure we need this one.

Reply:
Andy recommended that we have a plain easy responsive image display for the use of ckeditor images. I thought it would be a good catchall for instances like that and included it.

Reply:
Great!